### PR TITLE
Pillow image module

### LINKED ISSUE #4621: [Concept Entry] Python pillow - image module
### Reason/inspiration (optional)

Create a new concept entry for the `Image Module` of `Pillow` library and include the `https://pillow.readthedocs.io/en/stable/reference/Image.html` in general. It should go under the path: `docs/content/pillow/concepts/image/image.md`
It can have the following:
- A short description of the concept
- ## Syntax for importing the concept

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entrty for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by MattiasBaldi**
### Description

Created a concept entry for the image module in python pillow. Explained how to import it and what it can be used for using: https://pillow.readthedocs.io/en/stable/reference/Image.html

### Issue Solved

Closes #4621

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by mamtawardhani** [comment]
Perfect @MattiasBaldi! See, you did it 😁 
Will review it in a couple of hours and add some suggestions, just review and modify those by today if possible.
Thanks!

File: content/pillow/concepts/image/imagemodule.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+In Python, the Pillow library is an advanced image processing tool designed to simplify the manipulation and editing of images by providing a wide range of easy-to-use functions for tasks such as opening, saving, filtering, enhancing, and transforming images, allowing users to perform complex image processing with minimal code.
```
Comment (mamtawardhani) [inline content/pillow/concepts/image/imagemodule.md:1]:
```suggestion
---
Title: 'Image'
Description: 'The Image module in Pillow is used for image processing tasks in Python, offering functionalities like opening, editing, and saving images in different formats, as well as applying filters, resizing, and accessing pixel data.'
Subjects:
  - 'Computer Science'
  - 'Data Science'
  - 'Data Visualization'
Tags:
  - 'Images'
  - 'Modules'
  - 'Pilow'
CatalogContent:
  - 'learn-python-3'
  - 'paths/computer-science'
  - 'paths/data-science'
  - 'paths/data-science-foundations'
---

The **Image** module in Pillow, a popular Python library for image processing, provides a comprehensive suite of tools for opening, creating, manipulating, and saving images in various formats. It allows users to perform a wide range of operations such as resizing, cropping, rotating, and flipping images. Additionally, the module supports converting images between different modes (e.g., RGB to grayscale) and accessing or modifying individual pixels, making it a powerful and versatile tool for handling image data in Python applications.
```

File: content/pillow/concepts/image/imagemodule.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+## Syntax
+```
+from PIL import Image
+```
```
Comment (mamtawardhani) [inline content/pillow/concepts/image/imagemodule.md:6]:
```suggestion
```

This statement imports the Image module specifically from the Pillow library, which is a fork of the Python Imaging Library (PIL).
```

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @MattiasBaldi, thank you for contributing to the Codecademy Docs! The entry is nicely written!

I've just suggested the meta-data at the start, please commit that suggestion, and also, could you just change the file name, because the required bath is `docs/content/pillow/concepts/image/image.md`, but you have named the file as `imagemodule.md`. 
Could you please do the above two tasks at your earliest convenience? 
Thank you! 😄

**Review by mamtawardhani — APPROVED** [review]
@MattiasBaldi Thank you so much for making the required changes! 😄 

Looks good for a second review!

**Review by Sriparno08 — APPROVED** [review]
Looks good! 👍

**Pull request merged**